site/content/en/reference/cfg/search/_index.md

```diff
@@ -0,0 +1,195 @@
+---
+title: "Search"
+linkTitle: "search"
+weight: 4
+type: docs
+description: >
+  Search and optionally replace fields across all resources.
+---
+
+<!--mdtogo:Short
+    Search and optionally replace fields across all resources.
+-->
+
+{{% pageinfo color="primary" %}}
+
+This feature is under development. Please set environment variable
```

Comment:
In this branch we should make this functionality available without requiring the env variable.

Reply:
Done.